models/post.js

```diff
@@ -3,7 +3,6 @@ const mongoose = require("mongoose");
 const PostSchema = new mongoose.Schema({
   message: String,
   user_id: {type: mongoose.Schema.Types.ObjectId, ref: 'User'},
-  username: String,
```

Comment:
this is unnecessary because we populate from user_id to get the username of the post's creator